### PR TITLE
Display extension versions in 'about' screen

### PR DESCRIPTION
**Pull request by sammacbeth**
Show the version of embedded extensions in settings -> about

### CODE REVIEW

File: app/src/main/java/org/mozilla/reference/browser/settings/AboutFragment.kt [hunk 1]
```diff
@@ -31,6 +32,26 @@ class AboutFragment : Fragment() {
...
+                }
+                reader.skipValue()
+            }
+            reader.endObject()
+            return ""
+        } finally {
+            reader.close()
+            stream?.close()
+        }
+    }
```
Comment (spacifici) [inline app/src/main/java/org/mozilla/reference/browser/settings/AboutFragment.kt:53]: Cool! But, should it be not a `suspend` function? Doesn't it execute on the main thread?

Comment (sammacbeth) [reply app/src/main/java/org/mozilla/reference/browser/settings/AboutFragment.kt:53]: Yes, this is on main thread. What's the best way to use a suspend function in a Fragment? Which coroutine scope should be used and how to make `onViewCreated` async? Are there some examples of this pattern somewhere?

Comment (spacifici) [reply app/src/main/java/org/mozilla/reference/browser/settings/AboutFragment.kt:53]: I think that actually we can generate it at compile time and add it as a R.string. Let me check.

Comment (spacifici) [reply app/src/main/java/org/mozilla/reference/browser/settings/AboutFragment.kt:53]: Something like this: https://github.com/sammacbeth/daisy/pull/1

**Pull request merged**